06.wc/wc.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require 'optparse'
+
+def string_count(file)
+  file.count("\n")
+end
+
+def word_count(file)
+  word = file.split(/\s+/)
+  word.size
+end
+
+def bite_size(file)
+  file.bytesize
+end
+
+# 処理開始
+# パイプがあるかどうかを判定
+if File.pipe?($stdin) == false
```

Comment:
rubyの式というのは必ず`true`か`false`のどちらかとして評価されます。
ですのでifなどの条件式で`true`や`false`と比較するシチュエーションは基本的に存在しなくなります。

```suggestion
unless File.pipe?($stdin)
```

ここだったらこうがいいかもです〜